bitnami/contour/templates/default-backend/ingress.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.defaultBackend.enabled }}
+apiVersion: {{ include "common.capabilities.ingress.apiVersion" . }}
```

Comment:
This is throwing an error on my end as there is no `.Values.ingress.apiVersion` in `values.yaml`.

Reply:
Ahh.. this is common v1.2.0 :)
Fixed - defined the values.